service/src/test/java/org/jboss/sbomer/service/test/unit/feature/sbom/errata/ReleaseAdvisoryEventsListenerTest.java

```diff
@@ -729,6 +731,7 @@ void testTextOnlyReleaseErrataWithManifests() throws IOException {
                 .withReleaseGenerations(pvToGenerations)
                 .build();
         listenerTextOnlyManifests.onReleaseAdvisoryEvent(event);
+        event.getReleaseGenerations().values().forEach(request -> assertNotEquals(RequestEventStatus.FAILED, request.getRequest().getEventStatus()));
```

Comment:
Is this correct? It seems the status is not `SUCCESS` but `IN_PROGRESS`, so we may check for that instead.

The previous code wasn't really checking anything (the method does not throw any exceptions, so it's unlikely to fail even if it doesn't work properly).

Reply:
You have a point. Sonar complains it is a test method with no assertions. Here I was checking `event and not `listenerTextOnlyManifests`, so at least it's not duplicating any of the other assertions in the listener.